shell/common/persistent_cache_unittests.cc

```diff
@@ -204,5 +205,35 @@ TEST_F(ShellTest, CanLoadSkSLsFromAsset) {
   fml::UnlinkFile(asset_dir.fd(), PersistentCache::kAssetFileName);
 }
 
+TEST_F(ShellTest, CanRemoveOldPersistentCache) {
+  fml::ScopedTemporaryDirectory base_dir;
+
+  fml::CreateDirectory(base_dir.fd(),
```

Comment:
Assert that you actually got the FD here.

Reply:
Done.